cirq/google/op_deserializer.py

```diff
@@ -31,6 +36,34 @@
     import cirq
 
 
+class OpDeserializer(abc.ABC):
+    """Generic supertype for op deserializers."""
+
+    @property
+    @abc.abstractmethod
+    def serialized_id(self) -> str:
+        """Returns the string identifier for the resulting serialized object.
+
+        This value should reflect the internal_type of the serializer.
```

Comment:
I am not sure I understand what this sentence means.  What does "should reflect the internal_type" imply?
Same comment for deserializer.

Reply:
I must have misunderstood the IDs when typing this up (plus a copy-paste error in the deserializer comment). Updated to (hopefully) more accurately represent the behavior.